api/v1alpha1/workflow_helpers.go

```diff
@@ -50,6 +50,14 @@ const (
 	// which pinned container profile is being used.
 	PinnedContainerProfileLabelNameSpace = "nnf.cray.hpe.com/pinned_container_profile_namespace"
 
+	// PinnedDataMovementProfileLabelName is a label applied to NnfStorage objects to show
```

Comment:
Should this be NnfDataMovement?

Reply:
The other profile types (i.e. storage, container) aren't using `Nnf`, so I followed suit here.

Reply:
I meant s/NnfStorage/NnfDataMovement/

Reply:
Oh whoops. I'll fix that.

Reply:
Actually, these aren't even used since I've updated NnfDatamovement to use ObjectReferences to point the profile being used.

I think I can remove these altogether.